client/src/cli/accessTokenPath.ts

```diff
@@ -0,0 +1,23 @@
+import home_dir from "https://deno.land/x/dir@1.5.1/home_dir/mod.ts";
+import { Path } from "https://deno.land/x/path@v3.0.0/mod.ts";
+import { err, ok, Result } from "./deps.ts";
+
+export type AccessTokenError = { reason: string };
```

Comment:
not sure why not just a string here (I do like that you set a type for it)
we can always change it to an object later if we want

Reply:
Not sure as well...